src/igr-tests/meson.build

```diff
@@ -1,33 +1,31 @@
 # Included from top-level meson.build
 
-## Prepare integration tests
-run_command([
-    'cp',
-    '-r',
-    './',
-    meson.current_build_dir()
-    ],
-    check : true
-)
+igr_tests_env += [
+     'IGR_FUNCTIONS=@0@/igr_functions'.format(meson.current_source_dir()),
+     'TEMP=@0@'.format(meson.current_build_dir()),
+     'DEBUG=yes'
```

Comment:
Need more info! Which test, what is the dinit command line?

Reply:
Please debug issues like this before opening a PR (next time). I am working on other things and since it's your own changes causing the issue it's up to you to debug it.

You should probably check what command line is actually being passed to `dinit` in these cases as a start.

Reply:
(to be clear, I don't expect you to debug `dinit` itself, but at least figure out how it is being run that makes it behave this way)

Reply:
Seems like it cause by double-quoted `$QUIET`. Fixed.

Reply:
Right, `"$VAR"` expands to an empty argument (not no argument) if VAR is empty. However this does reveal a minor dinit bug: it should probably check for empty arguments and not try to start a service with no name.